src/TraceView.tsx

```diff
@@ -139,6 +156,7 @@ class TraceView extends Component<TraceViewProps> {
               : `${DOWN_ARROW} ${timeLabel} : ${span.operation}`;
           const startTS = span.timestamp.toMillis();
           const endTS = span.timestamp.toMillis() + span.duration / MILLISECOND;
+          const color = StringToColour(span.operation)
```

Comment:
British/Canadian spelling, haha

Reply:
damn, I tried to get rid of that.  I'll fix it.  But it's really hard to not write colour properly every time.